sagemaker-python-sdk/pytorch_batch_inference/sagemaker_batch_inference_torchserve.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    !wget https://awsbatchblog.s3.us-west-2.amazonaws.com/BERTSeqClassification.mar
</pre><p>Please use boto3 or awscli to get this file, not wget.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Hi @jkroll-aws, this is a publicly hosted file, could you provide and example of how to download this file using awscli/boto3</p>

Reply:
<pre class="ql-syntax" spellcheck="false">import boto3
s3 = boto3.client("s3")
s3.download_file("awsbatchblog", "BERTSeqClassification.mar", "BERTSeqClassification.mar")
</pre><p>The <code>download_file()</code>  parameters are &lt;bucket&gt;, &lt;prefix&gt;, &lt;local filename&gt;.</p>

Reply:
<p>The torchserve bucket can also be retrieved through AWS CLI: <code>aws s3 cp s3://torchserve/tar_gz_files/BERTSeqClassification.tar.gz .&nbsp;</code>  </p>

Reply:
Done, thanks